Patryk_Fraczek/Device/ZadanieAngleCalculation.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of the ZadanieAngleCalculation project
+#
+# 
+#
+# Distributed under the terms of the LGPL license.
+# See LICENSE.txt for more info.
+
+""" Calculate angle from double_scalar
+
+double_scalar from Tango_test to angle
+"""
+
+__all__ = ["ZadanieAngleCalculation", "main"]
+
+# PyTango imports
+import PyTango
+from PyTango import DebugIt
+from PyTango.server import run
+from PyTango.server import Device, DeviceMeta
+from PyTango.server import attribute, command
+from PyTango.server import class_property, device_property
+from PyTango import AttrQuality, AttrWriteType, DispLevel, DevState
+# Additional import
+# PROTECTED REGION ID(ZadanieAngleCalculation.additionnal_import) ENABLED START #
+from numpy import arcsin
+# PROTECTED REGION END #    //  ZadanieAngleCalculation.additionnal_import
+
+
+class ZadanieAngleCalculation(Device):
```

Comment:
Złą praktyką jest mieszanie polskiego i angielskiego w kodzie. Wiem, że to czepialstwo z mojej strony, ale dobre nawyki lepiej wyrabiać sobie jak najwcześniej!